Suppress warning messages from gsd-* plugins which occur during a 
gdm start/restart.  

Some distros also continue to see these warnings
 - https://www.centos.org/forums//viewtopic.php?p=291090

Guess is there is a race condition between gnome-session/gnome-shell starting
up and the gnome-settings-daemons plugins getting exec'ed because in debug
mode this doesn't happen

We have tried to tell the customer that these should be ignored if there 
is no functionality loss, but the customer is persistent about suppressing
these warnings, so turning them into debug messages instead.

Not suitable for upstream and needs to be evaluated/dropped when updating 
to 3.32 (with gnome-session and gnome-settings-daemon changes)

--- a/gnome-session/gsm-manager.c	2019-07-08 09:36:56.866563851 -0700
+++ b/gnome-session/gsm-manager.c	2019-07-08 09:35:57.041033913 -0700
@@ -623,7 +623,7 @@
           int         signal,
           GsmManager *manager)
 {
-        g_warning ("Application '%s' killed by signal %d", gsm_app_peek_app_id (app), signal);
+        g_debug ("Application '%s' killed by signal %d", gsm_app_peek_app_id (app), signal);

         if (gsm_app_get_registered (app) && gsm_app_peek_autorestart (app)) {
                 g_debug ("Component '%s' is autorestart, ignoring died signal",
@@ -649,7 +649,7 @@
             GsmManager *manager)
 {
         if (exit_code != 0)
-                g_warning ("App '%s' exited with code %d", gsm_app_peek_app_id (app), exit_code);
+                g_debug ("App '%s' exited with code %d", gsm_app_peek_app_id (app), exit_code);
         else
                 g_debug ("App %s exited successfully", gsm_app_peek_app_id (app));
 
--- a/gnome-session/main.c	2019-07-08 09:56:06.063584631 -0700
+++ b/gnome-session/main.c	2019-07-08 09:56:13.659881442 -0700
@@ -81,7 +81,7 @@
         GsmManager *manager = (GsmManager *)data;
 
         if (connection == NULL) {
-                g_warning ("Lost name on bus: %s", name);
+                g_debug ("Lost name on bus: %s", name);
                 gsm_fail_whale_dialog_we_failed (TRUE, TRUE, NULL);
         } else {
                 g_debug ("Calling name lost callback function");
